modules/client/src/test/java/org/apache/ignite/client/ClientKeyValueBinaryViewTest.java

```diff
@@ -330,4 +331,100 @@ public void testGetAndPut() {
         assertEquals("1", res2.stringValue(0));
         assertEquals("3", kvView.get(null, tupleKey(1L)).stringValue(0));
     }
+
+    @Test
+    public void testGetNullable() {
+        KeyValueView<Tuple, Tuple> kvView = defaultTable().keyValueView();
+
+        Tuple existingKey = tupleKey(DEFAULT_ID);
+        Tuple nonExistingKey = tupleKey(-1L);
+
+        kvView.put(null, existingKey, null);
+        kvView.remove(null, nonExistingKey);
+
+        NullableValue<Tuple> nullVal = kvView.getNullable(null, existingKey);
+        NullableValue<Tuple> missingVal = kvView.getNullable(null, nonExistingKey);
+
+        assertNull(missingVal);
+
+        assertNotNull(nullVal);
+        assertNotNull(nullVal.get());
```

Comment:
javadoc says the following:
`{@code getNullable(tx, key)} returns {@code Nullable.of(null)} after {@code put(tx, key, null)}.`

looks like the current behaviour is incorrect

Reply:
Thanks, test reworked.